src/pharmpy/model/external/nonmem/update.py

```diff
@@ -1458,19 +1459,37 @@ def update_estimation(control_stream, model):
             newrec = create_record(str(rec))
             control_stream = control_stream.insert_record(newrec)
 
-    old_cov = False
+    # Initiate old_cov
+    old_cov = None
     for est in old:
-        old_cov |= est.cov
-    new_cov = False
+        if est.cov is None:
+            old_cov = None
+        elif est.cov is not None and est.cov in EstimationStep.supported_covs:
```

Comment:
No need to check if in supported_covs since this is an invariant. If it is not None it has to be supported.